benchmark/synthetic/models/lstm_text_generation_nietzsche.py

```diff
@@ -0,0 +1,187 @@
+'''
+Original Model from https://github.com/keras-team/keras/blob/master/examples/lstm_text_generation.py
+Benchmark for a LSTM model.
+Script template modified from TensorFlow Benchmark repo:
+https://github.com/tensorflow/benchmarks/blob/keras-benchmarks/scripts/keras_benchmarks/models/lstm_benchmark.py
+'''
+from __future__ import print_function
+import keras
+from keras.utils import multi_gpu_model
+
+from models import timehistory
+
+from keras.callbacks import LambdaCallback
+from keras.models import Sequential
+from keras.layers import Dense, Activation
+from keras.layers import LSTM
+from keras.optimizers import RMSprop
+from keras.utils.data_utils import get_file
+import numpy as np
+import random
+import sys
+import io
+import re
```

Comment:
please organize imports correctly - https://www.python.org/dev/peps/pep-0008/#imports

Reply:
Thank-You for the link. Are you referring to organizing one import per line or import alphabetically? I have updated a code. Let me know if I misunderstood it. Thanks.

Reply:
No - I meant this order - 
Imports should be grouped in the following order:

standard library imports
related third party imports
local application/library specific imports